### PR TITLE
[SOAR-18032] Cylance Protect 1.5.3 Release

### PR DESCRIPTION
**Pull request by rmurray-r7**
### Description

Ticket: https://rapid7.atlassian.net/browse/SOAR-18032

Snyk: https://app.snyk.io/org/soar-insightconnect-new/project/9323440b-99c2-4671-8d49-6f50e1a25c95

Describe the proposed changes:

- #2912 

  - SDK Bump to 6.1.4
  - Bumped requirements.txt

### CODE REVIEW

File: plugins/cylance_protect/help.md [hunk 1]
```diff
@@ -31,13 +31,13 @@ You must create a Custom Application by following this procedure from the Cylanc
...
 The connection configuration accepts the following parameters:  
 
-|Name|Type|Default|Required|Description|Enum|Example|
-| :--- | :--- | :--- | :--- | :--- | :--- | :--- |
-|application_id|credential_secret_key|None|True|Application ID for CylancePROTECT instance|None|1abc234d-5efa-6789-bcde-0f1abcde23f5|
-|application_secret|credential_secret_key|None|True|Generated token that allows access to Cylance Resources|None|1abc234d-5efa-6789-bcde-0f1abcde23f5|
-|tenant_id|credential_secret_key|None|True|The unique tenant ID of the tenant that the device belongs to|None|1abc234d-5efa-6789-bcde-0f1abcde23f5|
-|url|string|https://protectapi.cylance.com|True|Web API URL|None|https://protectapi.cylance.com|
-  
+|Name|Type|Default|Required|Description|Enum|Example|Placeholder|Tooltip|
```
Comment (nmccullagh-r7) [inline plugins/cylance_protect/help.md:34]: Is this a plugin tooling bug? Might be worth manually removing these additions and making a ticket

File: plugins/cylance_protect/plugin.spec.yaml [hunk 2]
```diff
@@ -24,22 +29,25 @@ key_features:
   - Quarantine endpoints
   - Search threats
   - Delete assets
+links:
+  - "[BlackBerry CylancePROTECT](https://www.cylance.com)"
 requirements:
   - CylancePROTECT configured with an Custom Application
 references:
-  - "[BlackBerry CylancePROTECT](https://www.cylance.com)"
+  - "[BlackBerry CylancePROTECT API](https://documentation.securonix.com/bundle/securonix-on-prem-user-guide/page/content/active-deployment-guides/cylance-protect-api-kvp.htm)"
```
Comment (nmccullagh-r7) [inline plugins/cylance_protect/plugin.spec.yaml:37]: this doc reference doesn't seem correct

Comment (rmurray-r7) [reply plugins/cylance_protect/plugin.spec.yaml:37]: The reference links to the API docs for Cylance and the link refers the Cylance's website.

File: plugins/cylance_protect/help.md (hunk 1 above)
Comment (rmurray-r7) [reply plugins/cylance_protect/help.md:34]: [All of the prior snyk tickets (which have been merged) have this in the help.md](https://github.com/rapid7/insightconnect-plugins/pull/2910/files#diff-c9045ce4af3162c1cb15f5e8f36b945bcef925c9ac433c217b00f9940e419ca7L41). I am assuming as these plugins (that have snyk vulns) are relatively old, it's more likely refactoring to match the modern help.md. The example input and output fields remain the same so it should be ok.

File: plugins/cylance_protect/plugin.spec.yaml (hunk 2 above)
Comment (nmccullagh-r7) [reply plugins/cylance_protect/plugin.spec.yaml:37]: oh interesting i got a 404 when went to it, must have copied a bracket

**Review by igorski-r7 — APPROVED** [review]

**Review by nmccullagh-r7 — APPROVED** [review]

File: plugins/cylance_protect/plugin.spec.yaml (hunk 2 above)
Comment (dlaverty-r7) [reply plugins/cylance_protect/plugin.spec.yaml:37]: @rmurray-r7 as discussed, I think this should link to the Blackberry docs please.

**Pull request merged**